### PR TITLE
FEAT: Added push excitation

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
         is_time_domain : bool, optional
             Either if the input data is Time based or Frequency Based. Frequency based data are Mag/Phase (deg).
```
Comment (PipKat) [inline pyaedt/hfss.py:3611]:
```suggestion
            Whether the input data is time-based or frequency-based. Frequency based data are Mag/Phase (deg).
```
The second sentence here should be rewritten in plain English.

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
...
         x_scale : float, optional
-            Scaling factor for x axis.
+            Scaling factor for x axis. This argument will be ignored if the algoritmm will
+             identify the format from the file header.
```
Comment (PipKat) [inline pyaedt/hfss.py:3614]:
```suggestion
            Scaling factor for the x axis. This argument is ignored if the algorithm
             identifies the format from the file header.
```

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
...
         y_scale : float, optional
-            Scaling factor for y axis.
+            Scaling factor for y axis. This argument will be ignored if the algoritmm will
+             identify the format from the file header.
```
Comment (PipKat) [inline pyaedt/hfss.py:3617]:
```suggestion
            Scaling factor for the y axis. This argument is ignored if the algorithm
             identifies the format from the file header.
```

File: pyaedt/hfss.py [hunk 4]
```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
...
         impedance : float, optional
             Excitation impedance. Default is `50`.
         data_format : str, optional
-            Either `"Power"`, `"Current"` or `"Voltage"`.
+            Either `"Power"`, `"Current"` or `"Voltage"`. This argument will be ignored if the algoritmm will
+             identify the format from the file header.
```
Comment (PipKat) [inline pyaedt/hfss.py:3622]:
```suggestion
            Data format. Options are ``"Current"``, ``"Power"``, and ``"Voltage"``. This
            argument is ignored if the algoritmm identifies the format from the
            file header.
```

File: pyaedt/hfss.py [hunk 5]
```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
...
         encoding : str, optional
             Csv file encoding.
```
Comment (PipKat) [inline pyaedt/hfss.py:3624]:
```suggestion
            CSV file encoding.
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM
